.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "run_dir/static/nvd3"]
+	path = run_dir/static/nvd3
+	url = https://github.com/novus/nvd3.git
```

Comment:
Should we add something to the internal docs how to pull this properly when deploying?

Reply:
It will be a one time thing, so maybe in the section for setting up?

Reply:
This seems to already be in the readme.